rspace/src/main/scala/coop/rchain/rspace/RSpaceClosedException.scala

```diff
@@ -0,0 +1,4 @@
+package coop.rchain.rspace
+
+class RSpaceClosedException(str: String = "RSpace closed") extends IllegalStateException()
```

Comment:
I recently faced a problem when classes don't pass the error message to inherited exception's constructor so `Throwable::getMessage` returns `null`. Not sure if is a problem in your case.

Reply:
Exactly, my miss, fixed.